CHANGELOG.asciidoc

```diff
@@ -3,6 +3,17 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.11.1]]
+=== Beats version 8.11.1
+https://github.com/elastic/beats/compare/v8.11.0\...v8.11.1[View commits]
+
+==== Added
+
+*Affecting all Beats*
+
+- Setting environmental variable ELASTIC_NETINFO:false in Elastic Agent pod will disable the netinfo.enabled option of add_host_metadata processor
```

Comment:
@pierrehilbert is it correct? Looking at the commit `ELASTIC_NETINFO:false`:
 - libbeat is the one reading `ELASTIC_NETINFO`, therefore it'll affect the beats regardless of them running on docker or anywhere else.
 - It's a beats changelog, so no reason to be mentioning the Elastic Agent here
 - Setting `ELASTIC_NETINFO:false` in an in Elastic Agent pod will affect the beats not because the agent will forward that to the beats, but because the beats run inside the agent pod and therefore they will pick it up